share/functions/__fish_print_zypp_repos.fish

```diff
@@ -0,0 +1,3 @@
+function __fish_print_zypp_repos -d "Print repositories of zypper package manager"
```

Comment:
Since this is not used anywhere else, can you just stick it in the completion script?

Also I'd appreciate it if this used our `string` builtin instead.

Something like

```fish
for file in /etc/zypp/repos.d/*.repo
    string replace -rf '\[(.*)\]' '$1' <$file
end
```

(This will match something like `[bla]` and replace it with `bla`. The "-f" for "--filter" will only use lines where a replacement can be performed)